julia/Dockerfile

```
@@ -0,0 +1,6 @@
+FROM ustcmirror/base:alpine
+MAINTAINER Elliot Saba <staticfloat@gmail.com>
+RUN apk add --no-cache python3 git bash && \
```

Is it a hard dependency for bash? If not, probably just use busybox's shell

```suggestion
RUN apk add --no-cache python3 git && \
```